authentication.go

```diff
@@ -95,3 +104,16 @@ func (a *App) ValidateAuthToken(authToken string) (string, bool) {
 func (a *App) ValidateJWT(authToken string) (string, bool) {
 	return a.ValidateAuthToken(authToken)
 }
+
+func (a *App) getExpectedAudienceValue() (string, error) {
+	appInfo, err := a.GetApp()
```

Comment:
made these changes

Reply:
sorry there may have been some confusion. i meant that the network call that is being executed in `a.GetApp()` is being called every single time someone validates a JWT, which is unnecessary because the only value required from that call is the auth origin.

instead we could store the result of that network call in the constructor of the App or Auth structs and just reference the auth origin from there when validating JWTs.

Reply:
@ctran88 A network call for each validation isn't ideal, but caching it could cause issues if the auth origin changes or hosting is toggled on. The server using the SDK would require a restart to pick up the latest value.

This is a downside of using a mutable value as the audience claim, but it may be simpler for now to fetch it as needed and transition to a static app ID once the backend update is fully rolled out.

Reply:
would it be preferable to not validate at all since we haven't done it in the past?

if we push a release where the network call is being made every time that's big behavioral change for devs.

waiting for the backend update rollout and for existing tokens to fully expire could be hard to time with pushing another minor release to remove the network calls.

if we need to push our major before then, it would also cause us to have to hotfix the previous major

Reply:
Yeah good point. The feature with the JWT change is slated for release tomorrow morning, so it's simpler to assume that the rollout will be successful & just account for the app ID in the audience claim here.